integration/helpers/fixture.go

```diff
@@ -51,7 +51,8 @@ func NewFixture(t *testing.T) *Fixture {
 	require.NoError(t, err)
 
 	// Find AllocatePortsNum free listening ports to use.
-	fixture.Me, _ = user.Current()
+	fixture.Me, err = user.Current()
```

Comment:
Unrelated to the fix itself, but it bit me at some point, so I decided to fix it.